_posts/2019-01-21-workshop.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title:  "6th Spinal Cord MRI Workshop"
+title:  "Spinal Cord MRI Workshop@UCL"
```

Comment:
do you want a space between "Workshop" and "@UCL"?

Reply:
Ok